content/02.introduction.md

```diff
@@ -15,14 +15,13 @@ The Pediatric Brain Tumor Atlas (PBTA) initiative established in 2018 by the Chi
 Just as cooperation accelerates specimens and data sharing, collaboration among researchers including computational and bench scientists, clinicians, and pathologists is critical for rigorous genomic analysis.
 
 Although there has been significant progress elucidating genomic bases of pediatric brain tumor formation and progression, translating therapeutic agents to phase II or III clinical trials and subsequent FDA approvals have not kept pace.
-Within the last 20 years, the FDA has approved only seven targeted agents for treating pediatric brain tumors: mTOR inhibitor everolimus, for subependymal giant cell astrocytoma; anti-PD-1 immunotherapy pembrolizumab, for microsatellite instability–high or mismatch repair–deficient tumors; NTRK inhibitors larotrectinib and entrectinib, for tumors with an NTRK 1/2/3 gene fusions; MEK1/2 inhibitor selumetinib, for neurofibromatosis type 1 (NF1) and symptomatic, inoperable plexiform neurofibromas; and combination therapy MEK1/2 inhibitor trametinib and BRAF/CRAF inhibitor dabrafenib for unresectable or metastatic progressive tumors with BRAF V600E mutations [@url:https://www.fda.gov/about-fda/oncology-center-excellence/pediatric-oncology-drug-approvals].
-
-This is, in part, due to pharmaceutical company priorities and toxicity concerns, posing challenges for researchers to obtain new therapeutic agents for pediatric clinical trials.
+Within the last 20 years, the FDA has approved only seven targeted agents for treating pediatric brain tumors [@url:https://www.fda.gov/about-fda/oncology-center-excellence/pediatric-oncology-drug-approvals].
+This is, in part, due to pharmaceutical company priorities and toxicity concerns, which pose challenges for pediatric clinical trials.
```

Comment:
This sounds a bit vague now - how about:

"This is partly due to pharmaceutical company priorities, posing challenges for researchers to obtain therapeutic agents for pediatric clinical trials."